src/Common/src/Interop/Windows/mincore/Interop.VersionHelpers.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Kernel32)]
+        private static extern bool VerifyVersionInfo([In] ref OSVERSIONINFOEX lpVersionInfo, uint dwTypeMask, ulong dwlConditionMask);
+
+        [DllImport(Libraries.Kernel32)]
+        private static extern ulong VerSetConditionMask(ulong dwlConditionMask, uint dwTypeBitMask, byte dwConditionMask);
```

Comment:
When we were doing this in DNX using `GetVersion`, we had to ensure the dnx executable had a Windows manifest indicating it supported up to and including Windows 10. Do you know if that is still a limitation with this API? Mostly just want to know for information and I figure whoever wrote this P/Invoke will know better than I ;P.